src/components/ui/coreValues.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import FolderIcon from "@mui/icons-material/Folder";
+import values from "@/data/coreValuesData.json";
+
+const CoreValues: React.FC = () => {
+
+
+  return (
+    <div className="max-w-6xl mx-auto mt-10">
+      {values.map((value, index) => (
+        <div
+          key={index}
+          className="flex items-start py-6 border-b border-gray-300 last:border-0 gap-6"
+        >
+          {/* Folder Icon */}
+          <div className="flex-shrink-0 w-20">
+            <FolderIcon className="text-gray-500" fontSize="large" />
```

Comment:
this icon needs to be the outline Folder icon. See:
![Screenshot 2024-12-10 at 7 08 35 AM](https://github.com/user-attachments/assets/15098e42-f32d-411f-bba6-5808d5af095e)